src/main/java/org/torusresearch/torusutils/helpers/IsNewKeyResponse.java

```diff
@@ -0,0 +1,20 @@
+package org.torusresearch.torusutils.helpers;
+
+public class IsNewKeyResponse {
+
+    public boolean isNewKey;
+    public String publicKeyX;
+
+    public IsNewKeyResponse(boolean isNewKey, String publicKeyX) {
+        this.isNewKey = isNewKey;
+        this.publicKeyX = publicKeyX;
+    }
+
+    public boolean isNewKey() {
+        return isNewKey;
+    }
+
+    public String getPublicKeyX() {
+        return publicKeyX;
+    }
```

Comment:
These shouldn't be needed if the variables they relate to are public, either take these out or make what they refer to private.